src/components/Thrash/Thrash.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+
+import React from "react";
+import Styles from "./thrash.module.css";
+
+
+export default function Thrash() {
```

Comment:
Let's rename this component `Button`. And let's make it reusable by passing props to it:

type?: 'primary' | 'secondary';
icon?: ReactNode;
iconPosition?: 'left' | 'right' ;

Then you can set the default type to be 'primary' and the default iconPosition to be 'left'. Then you can use it like this:

<Button type='secondary' icon={trashIcon} />